lib/Chat/SystemMessage/Listener.php

```diff
@@ -213,6 +213,10 @@ protected function sendSystemGuestPermissionsMessage(RoomModifiedEvent $event):
 			return;
 		}
 
+		if ($event->getRoom()->isFederatedConversation()) {
```

Comment:
Should we move this to the `handle()` function so we always skip it, instead of adding it on each method over and over again? Basically the whole listener should never create a message if its a federated convo

Then we can also skip a potential integration test from my side :speak_no_evil: